server/main.py

```
@@ -0,0 +1,280 @@
+from fastapi import FastAPI, HTTPException
+from fastapi.responses import FileResponse
+from fastapi.middleware.cors import CORSMiddleware
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)